### ISSUE TITLE
[ENH] List "Interviews" as the 4th flap in "People"

### ISSUE THREAD

**Issue by harshnisar** [harshnisar/ml-india#110]
http://ml-india.org/people/

**Comment by bsinghpratap** [comment]
I don't think that fits into that page. This page is mainly for other ML folks to find people/labs who are working in the similar domain. 

It's better to have tabs in the blogs' page: 
- Interviews
- Meetups
- Conference Analysis
- Conference Coverages
etc.

**Comment by harshnisar** [comment]
Agreed. 

By tabs do you mean the same accordion flaps such as in people?

My only problem with accordion flaps is that Google punishes the page in it's listing because user doesn't find what he searched for on the first page. 

What other options do we have?
